snuba/snuba_migrations/generic_metrics/0014_distribution_add_options.py

```diff
@@ -0,0 +1,82 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    storage_set_key = StorageSetKey.GENERIC_METRICS_DISTRIBUTIONS
+
+    local_table_name = "generic_metric_distributions_raw_local"
+    dist_table_name = "generic_metric_distributions_raw_dist"
+
+    columns = [
+        (
+            Column("enable_histogram", UInt(8, MigrationModifiers(default=str("0")))),
+            "granularities",
+        ),
+        (
+            Column(
+                "decasecond_retention_days",
+                UInt(8, MigrationModifiers(default=str("retention_days"))),
+            ),
+            "enable_histogram",
+        ),
+        (
+            Column(
+                "min_retention_days",
+                UInt(8, MigrationModifiers(default=str("retention_days"))),
+            ),
+            "decasecond_retention_days",
+        ),
+        (
+            Column(
+                "hr_retention_days",
+                UInt(8, MigrationModifiers(default=str("retention_days"))),
+            ),
+            "min_retention_days",
+        ),
+        (
+            Column(
+                "day_retention_days",
+                UInt(8, MigrationModifiers(default=str("retention_days"))),
+            ),
+            "min_retention_days",
```

Comment:
This should probably be `hr_retention_days`. `min_retention_days` is already used above.

Reply:
Fixed, good catch